<launch>
  <arg name="device_name" default="/dev/ttyUSB0"/>
  <arg name="baud_rate"   default="1000000"/>
  <arg name="scan_range"  default="10"/>

  <param name="device_name" type="str" value="$(arg device_name)"/>
  <param name="baud_rate"   type="int" value="$(arg baud_rate)"/>
  <param name="scan_range"  type="int" value="$(arg scan_range)"/>

  <node name="print_registers"
  	    type="print_registers"
  	    pkg="cmu_16662_robot_utils"
  	    output="screen"/>
</launch>
